scripts/azureml-assets/azureml/assets/config.py

```diff
@@ -179,42 +182,42 @@ def _validate(self):
         Raises:
             ValidationException: If validation failed.
         """
-        Config._validate_exists('name', self.name)
-        Config._validate_exists('version', self.version)
+        Config._validate_exists("name", self.name)

```

Comment:
>("n [](http://example.com/codeflow?start=31&length=3)

(nit) These could be in single quotes may be?

Reply:
The double quotes seem to be a formatting change. All the properties in the config file are in double quotes.